lib/eco_service_web/live/eco_service_live/edit_community_component.ex

```diff
@@ -0,0 +1,39 @@
+defmodule EcoServiceWeb.EcoServiceLive.EditCommunityComponent do
+  use EcoServiceWeb, :live_component
+
+  alias EcoService.EcoServiceContext
+
+  def render(assigns) do
+    ~H"""
+      <div>
+        <.simple_form :let={form} for={%{}} phx-target={@myself} phx-submit="update-community">
+          <.input field={form[:name]} label="Community Name" value={@community.name}/>
+          <.input field={form[:location_area_zone]} label="Location" value={@community.location_area_zone}/>
+          <.button>Save</.button>
+        </.simple_form>
+      </div>
+    """
+  end
+
+  def update(assigns, socket) do
+    community = EcoServiceContext.get_community_by_id(assigns.community_id)
+
+    {:ok,
+    socket
+    |> assign(:community, community)
```

Comment:
Using just `|>` operator for a function is generally not considered as a good practice. There is a way that you can write this without it. Can you guess?

Reply:
assign(socket, :community, community) ?